src/libs/actions/Report.js

```diff
@@ -1489,8 +1489,8 @@ function navigateToConciergeChat() {
         return;
     }
 
-    Navigation.navigate(ROUTES.getReportRoute(conciergeChatReportID));
     Navigation.closeDrawer();
+    Navigation.navigate(ROUTES.getReportRoute(conciergeChatReportID));
```

Comment:
I'm noticing that now that these are swapped, we see the 'Get assistance' page close as the chat gets opened with concierge. This creates two sliding pages at once, which feels a bit strange. Did you have a specific reason in mind for this change?

https://user-images.githubusercontent.com/49007721/147994350-1142b1e1-6d85-4d17-a424-c7e7f4acb6f1.mov

Reply:
Ah, yes. The double animation is indeed a bit jarring. 

The reason I flipped the order was because in the original order there was a bug where it would only actually navigate to the concierge chat every other time you tapped it for some reason.

Reply:
Oh... interesting.
That seems like something we should investigate. Perhaps it'd be best to make a new issue for that. 

Swapping the order feels like a workaround rather than a fix. I'd vote to keep the broken behavior around so that it sticks out and we address it.

Reply:
>in the original order there was a bug where it would only actually navigate to the concierge chat every other time you tapped it for some reason

Are we investigating this anywhere else? Should we mention that the order is significant in the code? Looking at these two lines and nothing about them tells me they are arranged in that way to prevent a bug.

Reply:
Ok, that's good feedback. I reported it as a bug [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1641323776029200) and I'll revert this change so that it can be handled separately.

Reply:
Seems there's actually already an issue here: https://github.com/Expensify/App/issues/6908